packages/flutter/test/painting/image_provider_test.dart

```diff
@@ -57,55 +57,6 @@ void main() {
     expect(await caughtError.future, true);
   });
 
-  test('resolve sync errors will be caught', () async {
```

Comment:
These tests use mocks so unfortunately it is not representative. I did write a devicelab test that should continue passing